.orchestra/config/components/toolchain/lib/gdb.lib.yml

```diff
@@ -42,22 +43,47 @@ configure: |
     --disable-gold \
     --disable-gprof \
     --disable-ld \
-    --disable-gdbserver \
+    --enable-gdbserver \
     --enable-64-bit-bfd \
     --disable-install-libbfd \
     --disable-install-libiberty \
     --without-guile \
     --with-system-readline \
     --enable-nls \
-    --enable-tui=no \
+    --enable-tui \
     --with-mpfr="$ORCHESTRA_ROOT" \
     --without-included-gettext \
     --disable-shared \
     --enable-obsolete \
     --without-stage1-ldflags \
+    --with-python="$ORCHESTRA_ROOT/bin/python" \
+    --with-system-gdbinit="$ORCHESTRA_ROOT/(@= triple @)/etc/gdb/gdbinit" \
```

Comment:
These are most likely problematic and non-portable across binary-archives.
I think we should not specify them.

Reply:
It actually appears to be portable:

```
$ sudo mount --bind /shared/orchestra/root/ /mnt/

$ cat /mnt/x86_64-pc-linux-gnu/etc/gdb/gdbinit
set pagination off
set disassembly-flavor intel
p "hello world"

$ /mnt/x86_64-pc-linux-gnu/x86_64-pc-linux-gnu/binutils-bin/12.1/gdb -q
$1 = "hello world"
(gdb) pi
>>> import sys
>>> sys.executable
'/mnt/bin/python'
>>> sys.path
['/mnt/share/binutils-data/x86_64-pc-linux-gnu/12.1/gdb/python', '/mnt/lib/python310.zip', '/mnt/lib/python3.10', '/mnt/lib/python3.10/lib-dynload', '/mnt/lib/python3.10/site-packages']
>>>
(gdb) quit
```

Reply:
Eh no, it's a bug in the orchestra design, can't accept it.
For your use case, just put the stuff in `~/.gdbinit`.

Reply:
What about python?

Reply:
I just read the `configure` source and it appears that the GDB authors thought about this problem already and made it so that if these directories are under `$PREFIX` they are automatically made relative, which is why my example works.

Let me see if there's a way to make it more obvious that this is the case.

Reply:
Ah that's very nice! A comment will do.